app/src/main/java/io/horizontalsystems/bankwallet/modules/transactions/transactionInfo/TransactionInfoInteractor.kt

```diff
@@ -7,20 +7,25 @@ import io.horizontalsystems.bankwallet.core.providers.FeeCoinProvider
 import io.horizontalsystems.bankwallet.entities.Coin
 import io.horizontalsystems.bankwallet.entities.CurrencyValue
 import io.horizontalsystems.bankwallet.entities.LastBlockInfo
+import io.horizontalsystems.core.IBuildConfigProvider
 import io.horizontalsystems.core.ICurrencyManager
 
 class TransactionInfoInteractor(
         private var clipboardManager: IClipboardManager,
         private val adapter: ITransactionsAdapter,
         private val xRateManager: IRateManager,
         private val currencyManager: ICurrencyManager,
-        private val feeCoinProvider: FeeCoinProvider
+        private val feeCoinProvider: FeeCoinProvider,
+        private val buildConfigProvider: IBuildConfigProvider
 ) : TransactionInfoModule.Interactor {
     var delegate: TransactionInfoModule.InteractorDelegate? = null
 
     override val lastBlockInfo: LastBlockInfo?
         get() = adapter.lastBlockInfo
 
+    override val testMode: Boolean
+        get() = buildConfigProvider.testMode
+
```

Comment:
Is this field can dynamically be changed? Can we assign its value directly without getter?